src/Dhall/Main.hs

```diff
@@ -95,23 +96,17 @@ parseOptions :: Parser Options
 parseOptions =
         Options
     <$> parseMode
-    <*> parseExplain
-    <*> parsePlain
+    <*> switch "explain" "Explain error messages in more detail"
+    <*> switch "plain" "Disable syntax highlighting"
+    <*> switch "ascii" "Format code using only ASCII characters"
```

Comment:
`Text` literals are still formatted in `Unicode` right?

Reply:
Yes

Reply:
It didn't seem obvious to me from the description. But I guess you wouldn't really be able to make the `Text` literals `ascii` only anyway.
If I'm the only one who thought the description was weird leave it like as it is! :smile:

Reply:
Would "Format code using only ASCII syntax" have been more clear?

Reply:
How about "Format code using ASCII syntax for Dhall built-ins"? Or is that too long?

Reply:
The issue with "built-in" is that it means either built-in expressions like `Natural/even` or `Optional/fold` or built-in operators like `+`, but not syntactic elements like `λ`/`→`

Reply:
In that case your version with "syntax" instead of "characters" is better. Sorry for being so picky about the wording. 😅